marklogic-data-hub-central/ui/src/util/environment.tsx

```diff
@@ -1,16 +1,15 @@
 import axios from 'axios';
 
 const defaultEnv = {
-    projectName: '',
+    serviceName: '',
     dataHubVersion: '',
     markLogicVersion: ''
 }
 
 export function setEnvironment()  {
-    axios.get('/api/environment/project-info')
+    axios.get('/api/environment/systemInfo')
```

Comment:
One thought I've had for the future is to standardize on (I think) "system" instead of using "environment" and "system" interchangeably, as having those separate terms raises the question of - What does each one mean? "serviceName" adds a little more confusion to because it implies there's the concept of a "service" that is separate from "system". Really, system = environment = service, so in the future, I think we'll simplify the naming here.